bin/docserv

```diff
@@ -95,7 +92,7 @@ class Deliverable:
         tmp_build_target = tempfile.mkdtemp()
         dc_full_path = os.path.join(self.local_repo_build_dir, self.dc_file)
         commands[n] = {}
-        commands[n]['cmd'] = "/usr/bin/d2d_runner -v=0 -x %s -d %s -o %s -i %s -f %s %s" % (
+        commands[n]['cmd'] = "/usr/bin/d2d_runner -v=0 -x=%s -d=%s -o=%s -i=%s -f=%s %s" % (
```

Comment:
for my information: what are you doing with the output here? are we saving the log somewhere?